<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProSetPage</class>
 <widget class="QWizardPage" name="ProSetPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="2">
      <widget class="QLineEdit" name="lePath"/>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="leName"/>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="lblPath">
       <property name="text">
        <string>Path:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>name:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="btnBrowse">
       <property name="text">
        <string>browse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="lblTips">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
